tsconfig.json

```diff
@@ -1,17 +1,13 @@
 {
   "compilerOptions": {
-    "declaration": true,
     "esModuleInterop": true,
     "inlineSources": true,
     "lib": ["ES2020"],
     "module": "CommonJS",
-    "moduleResolution": "Node",
-    "outDir": "dist",
-    "rootDir": "src",
+    "moduleResolution": "node",
+    "noEmit": true,
     "sourceMap": true,
     "strict": true,
-    "target": "ES2017"
-  },
-  "exclude": ["./src/**/*.test.ts"],
```

Comment:
Maybe worth either preserving a broader `include` rule, or adding an `exclude` for `dist` here, for the reasons discussed here: https://github.com/MetaMask/eth-block-tracker/pull/103#discussion_r858878740